.changes/unreleased/tinymce-TINY-11916-2025-03-26.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: ContextFormSizeInput's buttons order.
```

Comment:
Please describe what changed in a little more detail.

Reply:
@TheSpyder check the new version